### PR TITLE
apm launch file update as suggested by Ajay

### PR DESCRIPTION
**Pull request by hildebrandt-carl**
+ Changed package name from pixhawk_defaults to freyja_configfiles
+ Changed vicon_object to vicon_path

### CODE REVIEW

**Comment by ajshank** [comment]
@hildebrandt-carl Perhaps better to name it `vicon_topic` as a better choice of name?

File: freyja_controller.launch [hunk 1]
```diff
@@ -8,9 +8,9 @@
 	-->
 
 	<!-- allowed autopilots: "asctec" or "arducopter" -->
-	<!-- must provide vicon_object -->
+	<!-- must provide vicon_path -->
 	<arg name="autopilot" default="arducopter"/>
-	<arg name="vicon_object"/>
+	<arg name="vicon_path"/>
```
Comment (ajshank) [inline freyja_controller.launch:13]: This could be `vicon_topic`, I vote ..

Comment (hildebrandt-carl) [reply freyja_controller.launch:13]: Good idea

**Comment by hildebrandt-carl** [comment]
Good idea, vicon_topic is more general and understandable.

**Review by ajshank — APPROVED** [review]

**Pull request merged**